kubevela.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: kubevela
+  version: 1.6.7
+  epoch: 0
+  description: KubeVela is a modern application delivery platform that makes deploying and operating applications across today's hybrid, multi-cloud environments easier, faster and more reliable
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+      - git
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/kubevela/kubevela/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 0b4d43518692e5c758148d1f8d78fe98f3134082cb4b3b49ca1200e3e44e8ade
+  - runs: |
+      GOLDFLAGS="-s -w
+        -X github.com/oam-dev/kubevela/version.VelaVersion=${{package.version}}
+        -X github.com/oam-dev/kubevela/version.GitRevision=git-$(git rev-parse --short HEAD)
```

Comment:
this shouldn't work because the tarball you download is not a git directory

Reply:
Very good point! Any advices for this case? Maybe I could set: `github.com/oam-dev/kubevela/version.GitRevision=Wolfi` etc.

Reply:
@Dentrax i don't know about this either. maybe @rawlingsj can advise?

Reply:
I'm thinking if its worth to add a lint check for this case: _"given download is tarball and `$(git` shell command is contains in `runs` field."_ One pitfall of this is that we couldn't catch if `make` is used, and it calls `git` command under the hood.